sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -86,7 +86,7 @@ export interface Entity {
 }
 
 // @public
-export type ErrorCodeValue = 'invalidRequest' | 'invalidArgument' | 'internalServerError' | 'serviceUnavailable';
+export type ErrorCode = "InvalidRequest" | "InvalidArgument" | "InternalServerError" | "ServiceUnavailable" | "InvalidParameterValue" | "InvalidRequestBodyFormat" | "EmptyRequest" | "MissingInputRecords" | "InvalidDocument" | "ModelVersionIncorrect" | "InvalidDocumentBatch" | "UnsupportedLanguageCode" | "InvalidCountryHint";
```

Comment:
idk much about JS, but what happens if the service adds a new `ErrorCode` ?

Reply:
I addressed this in the latest commit. This line was only meant to be a workaround for the swagger bug but I have patched it in the generated code.